tika-parent/pom.xml

```diff
@@ -730,6 +723,16 @@
         <artifactId>commons-math3</artifactId>
         <version>${commons.math3.version}</version>
       </dependency>
+      <dependency>
+        <groupId>org.apache.cxf</groupId>
+        <artifactId>cxf-rt-rs-client</artifactId>
+        <version>${cxf.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>org.apache.cxf</groupId>
+        <artifactId>cxf-rt-frontend-jaxrs</artifactId>
+        <version>${cxf.version}</version>
+      </dependency>
```

Comment:
This could be simplified:
```xml
      <dependency>
        <groupId>org.apache.cxf</groupId>
        <artifactId>cxf-bom</artifactId>
        <version>${cxf.version}</version>
        <type>pom</type>
        <scope>import</scope>
      </dependency>
```